TestPress.xcodeproj/xcshareddata/xcschemes/TestPress.xcscheme

```diff
@@ -26,7 +26,16 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
-      shouldUseLaunchSchemeArgsEnv = "YES">
+      shouldUseLaunchSchemeArgsEnv = "NO">
+      <MacroExpansion>
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "3F55172C2861780B002CF097"
+            BuildableName = "TestPressTests.xctest"
+            BlueprintName = "TestPressTests"
+            ReferencedContainer = "container:TestPress.xcodeproj">
+         </BuildableReference>
+      </MacroExpansion>
```

Comment:
This is a change in the Test part of the scheme that [some folks suggested](https://stackoverflow.com/a/42496856/809944) would have helped with getting access to the environment variables. I didn't help in our case, though. I've left it here for reference only.

<img width="926" alt="Screen Shot 2022-06-21 at 4 49 30 pm" src="https://user-images.githubusercontent.com/1218433/174734829-4150861b-965a-478d-aee1-8869e114d8a8.png">